src/edu/csus/ecs/pc2/shadow/ShadowScoreboardComparator.java

```diff
@@ -0,0 +1,391 @@
+package edu.csus.ecs.pc2.shadow;
+
+import java.io.IOException;
+import java.util.ArrayList;
+import java.util.Collections;
+import java.util.Comparator;
+import java.util.HashMap;
+import java.util.List;
+import java.util.Map;
+import java.util.PriorityQueue;
+
+import com.fasterxml.jackson.core.type.TypeReference;
+import com.fasterxml.jackson.databind.ObjectMapper;
+
+import edu.csus.ecs.pc2.core.log.Log;
+import edu.csus.ecs.pc2.core.standings.json.ScoreboardJsonModel;
+import edu.csus.ecs.pc2.core.standings.json.Team;
+import edu.csus.ecs.pc2.core.standings.json.TeamScoreRow;
+import edu.csus.ecs.pc2.ui.ShadowCompareScoreboardPane.ScoreboardType;
+
+public class ShadowScoreboardComparator {
```

Comment:
Rename this class (ShadowScoreboardComparer ?  ShadowScoreboardUtilityy?)    There are over 10 implements Comparator classes that implement implements java.util.Comparator and are used with sort() to sort.  
This class name implies that this class is a java.util.Comparator rather than a class that returns a list of comparisons.

Reply:
Done.